### DISCUSSION TITLE
登录

### DISCUSSION THREAD

**Discussion by Orangered81** [share121/inter-knot#5385] [绝区零]
怎么在Windows端登陆啊  
左上角显示不出头像

**Comment by FAIR-MOONLIGHT** [comment]
登录方法相同，随便进行一个需要登录才能进行的操作，然后会弹出登录链接

**Comment by Orangered81** [comment]
解决了，是在“我的”里面选择用户接口而非公共接口（另外一个帖子里面看到的，不过谢谢大佬）

**Reply by Robin-txdy** [reply]
我点了用户接口，但他让我弄扩展引擎，怎么弄啊

**Reply by Orangered81** [reply]
不造啊（拓展引擎事甚么<img src="https://bbs-static.miyoushe.com/static/2024/03/21/438ac4c6210e00a8f94a3a1eb13eb55e_7655048688282650889.png" alt="丽都漫步-本" width="75" >
